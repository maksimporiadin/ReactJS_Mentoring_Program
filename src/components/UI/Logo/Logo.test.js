import 'jsdom-global/register';
import React from 'react';
import { Logo } from './index';
import { shallow } from 'enzyme';

describe('Logo', () => {
    let wrapper;

    it('is renders', () => {
        wrapper = shallow(<Logo  />);
        expect(wrapper).toMatchSnapshot();
    });
});