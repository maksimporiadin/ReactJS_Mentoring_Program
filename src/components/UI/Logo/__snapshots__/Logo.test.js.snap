// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo is renders 1`] = `
<span
  className="logo-name"
>
  Netï¬‚ixroulette
</span>
`;
